### PR TITLE
Update support-request.yml

### PR DESCRIPTION
**Pull request by pattacini**
This PR adds up a further optional text area to the template to let the user quantify the impact of the issue on his/her workflow.

---

cc @maggia80 @Fabrizio69 @DanielePucci

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support-request.yml [hunk 1]
```diff
@@ -32,3 +32,7 @@ body:
   attributes:
     label: Additional context
     description: Please attach here any material you may consider useful (pictures, screenshots, logs, etc.).
+- type: textarea
+  attributes:
+    label: How does it affect me?
+    description: Additional field where you can quantify the impact of the issue on your workflow.
```
Comment (pattacini) [inline .github/ISSUE_TEMPLATE/support-request.yml:38]:
```suggestion
    description: Optional field where you can quantify the impact of the issue on your workflow.
```

File: .github/ISSUE_TEMPLATE/support-request.yml [hunk 2]
```diff
@@ -32,3 +32,7 @@ body:
   attributes:
     label: Additional context
     description: Please attach here any material you may consider useful (pictures, screenshots, logs, etc.).
+- type: textarea
+  attributes:
+    label: How does it affect me?
```
Comment (S-Dafarra) [inline .github/ISSUE_TEMPLATE/support-request.yml:37]: I was wondering if ``me`` or ``you`` is the most appropriate. For the person opening the issue, the question "How does it affect me?" might look like "How does it affect icub-tech?"

Comment (pattacini) [reply .github/ISSUE_TEMPLATE/support-request.yml:37]: Good point!
Let me amend it!

Comment (pattacini) [inline .github/ISSUE_TEMPLATE/support-request.yml:37]:
```suggestion
    label: How does it affect you?
```

`You` is coherent with the rest of the text.

**Review by gsisinna — APPROVED** [review]
For me it sounds good!

**Pull request merged**